integration-tests/docker-sws-api/api/build/appsettings.inttest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Elasticsearch": {
+    "Servers": "http://elasticsearch:9200",
+    "Userid": "",
+    "Password": "",
+    "MaximumRetries": 5,
+    "EnableDebugging": true
+  },
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug"
```

Comment:
Please change to Warning.  The log output is already noisy and as the  API doesn't presently use Debug, all that will show up is internal messages from the .Net framework.